lib/time/time.go

```diff
@@ -315,87 +333,87 @@ func (t Time) Binary(op syntax.Token, yV starlark.Value, side starlark.Side) (st
 }
 
 var timeMethods = map[string]builtinMethod{
-	"year":       timeyear,
-	"month":      timemonth,
-	"day":        timeday,
-	"hour":       timehour,
-	"minute":     timeminute,
-	"second":     timesecond,
-	"nanosecond": timenanosecond,
-	"unix":       timeunix,
-	"unix_nano":  timeunixnano,
+	"year":       timeYear,
+	"month":      timeMonth,
+	"day":        timeDay,
+	"hour":       timeHour,
+	"minute":     timeMinute,
+	"second":     timeSecond,
+	"nanosecond": timeNanosecond,
+	"unix":       timeUnix,
+	"unix_nano":  timeUnixNano,
 
-	"in_location": timein,
-	"format":      timeformat,
+	"in_location": timeIn,
+	"format":      timeFormat,
 }
```

Comment:
Just a question probably to @adonovan: Shouldn't these be attributes rather than builtins?